jenkins-integration/scripts/run_sbt.sh

```diff
@@ -3,14 +3,12 @@ set -x
 
 export BEAKER_TESTSUITE="${BEAKER_TESTSUITE:-beaker/gatling/20_run_sbt.rb}"
 export BEAKER_KEYFILE="~/.ssh/id_rsa-acceptance"
-export BEAKER_HELPER="beaker/helper.rb"
```

Comment:
Don't use this snowflake helper file.  Put code into either `jenkins-integration/lib/puppet/gatling/config.rb` or create new namespaces in that directory, and then explicitly reference them using `require 'puppet/gatling/config'` like normal Ruby.

We've never used this helper.rb (and it was empty) so I deleted in and all references to it.